java/clients/src/test/java/sleeper/clients/admin/TableNamesReportTest.java

```diff
@@ -48,11 +61,7 @@ void shouldPrintTableNamesReportWhenChosen() throws Exception {
                 .exitGetOutput();
 
         // Then
-        assertThat(output).isEqualTo(CLEAR_CONSOLE + MAIN_SCREEN + "\n\n" +
-                "Table Names\n" +
-                "----------------------------------\n" +
-                "test-table-1\n" +
-                "test-table-2\n" +
+        assertThat(output).isEqualTo(CLEAR_CONSOLE + MAIN_SCREEN + generateExpectedTableNamesOutput("test-table-1", "test-table-2") +
```

Comment:
Could we assert this the same way we were before please? We need to show the value we're asserting to be able to tell whether the test is correct.